pkg/sippyserver/server.go

```diff
@@ -866,18 +866,21 @@ func (s *Server) jsonReleasesReportFromDB(w http.ResponseWriter, _ *http.Request
 	response := apitype.Releases{
 		GADates: releaseloader.GADateMap,
 	}
-	releases, err := query.ReleasesFromDB(s.db)
+	releases, err := api.GetReleases(s.db, s.bigQueryClient)
 	if err != nil {
-		log.WithError(err).Error("error querying releases from db")
+		log.WithError(err).Error("error querying releases")
 		api.RespondWithJSON(http.StatusInternalServerError, w, map[string]interface{}{
 			"code":    http.StatusInternalServerError,
-			"message": "error querying releases from db",
+			"message": "error querying releases",
 		})
 		return
 	}
 
 	for _, release := range releases {
 		response.Releases = append(response.Releases, release.Release)
+		if release.GADate != nil {
+			response.GADates[release.Release] = *release.GADate
+		}
 	}
 
 	type LastUpdated struct {
```

Comment:
Current change is not totally independent from postgres due to the last updated query below. Should this be also modified?

Reply:
How about just not specifying it if there's no local db?   https://github.com/openshift/sippy/pull/1559/files#diff-768a5e234f25507b29e6156fd5af2bfb561aef7c7b19612e7e73ea4cb6be07e4R913

That's how I solved it for now, last updated is really only relevant to the data in postgres anyway

Reply:
I added the equivalent from BQ